src/parser/AstNode.ts

```diff
@@ -126,7 +133,7 @@ export abstract class Statement extends AstNode {
     /**
      * Annotations for this statement
      */
-    public annotations: AnnotationExpression[];
+    public annotations: AnnotationExpression[] = [];
```

Comment:
The only place that actually changes behavior based on whether `annotations` is `undefined` is a unit test in the parser (which I've updated to check whether it's an empty array instead.) All of the runtime code which uses `annotation` sets a default value of `[]` before using it. I've left those checks in just in case but I can remove them too.

Reply:
I rarely use getters so I'm not 100% certain, but I believe I could use a getter to ensure that the type isn't potentially `undefined` but also only create the array itself as needed. Even if it doesn't matter for performance now it doesn't hurt to keep things lean.